React/CoreModules/RCTActionSheetManager.mm

```diff
@@ -112,15 +112,15 @@ - (void)presentViewController:(UIViewController *)alertController
   UIColor *tintColor = [RCTConvert UIColor:options.tintColor() ? @(*options.tintColor()) : nil];
 
   if (controller == nil) {
-    RCTLogError(@"Tried to display action sheet but there is no application window. options: %@", @{
-      @"title" : title,
-      @"message" : message,
-      @"options" : buttons,
+    RCTLogError(@"Tried to display action sheet but there is no application window. options: %@", @{ /*  // [ TODO(macOS GH#774): nil check our dict values before inserting them or we may crash */
+      @"title" : title ?: [NSNull new],
```

Comment:
not `[NSNull null]` ?

Reply:
per https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/NumbersandValues/Articles/Null.html

Reply:
Functionally both seem to work but yea let's use `null`